Common/Core/RecoDecay.h

```diff
@@ -602,7 +604,7 @@ class RecoDecay
   /// \param stage  decay tree level; If different from 0, the particle itself will be added in the list in case it has no daughters.
   /// \note Final state is defined as particles from arrPDGFinal plus final daughters of any other decay branch.
   /// \note Antiparticles of particles in arrPDGFinal are accepted as well.
-  template <std::size_t N, typename T>
+  template <bool checkProcess = false, std::size_t N, typename T>
```

Comment:
Please add documentation of `checkProcess` before the `\param particle` line:
`/// \param checkProcess  switch to accept only decay daughters by checking the production process of MC particles`